tests/04_libvirt_controller.py

```diff
@@ -149,11 +149,7 @@ def test_02_start(self):
             'port': port,
         })
 
-    def test_03_start_no_spice_domain(self):
```

Comment:
It's not needed because now we are the ones that create the new graphics definition for that domain, so always will have an spice graphics adapter.